bin/process.py

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env python
 # coding=utf-8
+
+from __future__ import print_function
+import sys
+
+# Check for version
+if sys.version_info[0] <= 2:
+    print('Only works with Python 3. Please either change the first line of /usr/bin/yauml or make `python` points toward Python 3.')
```

Comment:
I don't think that the user should be changing the first line of the script as explained in my other comment.

If the user's `python` would eventually be pointing to something different from the OS maintainer's choice (changing the symbolic link), wouldn't it eventually lead to breaking the system's python package (for e.g. after a package upgrade)?